src/bramble/client/ProjectStats.js

```diff
@@ -0,0 +1,263 @@
+/* eslint
```

Comment:
I hear you.  I think it's really dangerous for us to start doing manual style changes within Brackets, because it will totally break our ability to merge with Adobe's upstream.  Any style changes we do have to be done automatically by a tool, for example, prettier.

If we want to do this in Brackets, my suggestion is as follows:
* Create a branch `adobe-upstream` which tracks Adobe's `master` branch
* Create a branch `adobe-upstream-pretty` which is `adobe-upstream` with auto-formatting applied.
* Figure out some kind of machinery to automatically keep the two above branches always up-to-date (maybe with git, maybe with travis, maybe with some git hook)
* Create some kind of grunt/npm task/git pre-commit hook/etc that runs our autoformatting on all code we land on `master`
* Do all of our future merges of `master` against `adobe-upstream-pretty`.  We don't do this very often.

If you're interested in doing this work, we can file a new bug.  I don't want to introduce separate lint rules per file. cc @Pomax